trtools/compareSTR/tests/test_compareSTR.py

```diff
@@ -173,6 +174,90 @@ def test_main(tmpdir, vcfdir):
     retcode = main(args)
     assert retcode == 1
 
+def test_no_comparable_records(tmpdir, vcfdir, capfd):
+    vcfcomp = os.path.join(vcfdir, "compareSTR_vcfs")
+    hipstr_vcf_1 = os.path.join(vcfcomp, "test_no_comparable_records_1.vcf.gz")
+    hipstr_vcf_2 = os.path.join(vcfcomp, "test_no_comparable_records_2.vcf.gz")
+
+    args = base_argparse(tmpdir)
+    args.vcf1 = hipstr_vcf_1
+    args.region = ""
+    args.vcf2 = hipstr_vcf_2
+
+    ret = main(args)
+    _, err = capfd.readouterr()
+    assert ret == 1
+    assert err == "No comparable records were found, exiting!\n"
+
+
+def test_better_comparability_calculation(tmpdir, vcfdir, capfd):
+    vcfcomp = os.path.join(vcfdir, "compareSTR_vcfs")
+
+    test_vcf_1 = os.path.join(vcfcomp, "test_better_comparability_calculation_1.vcf.gz")
+    test_vcf_2 = os.path.join(vcfcomp, "test_better_comparability_calculation_2.vcf.gz")
+
+    args = base_argparse(tmpdir)
+    args.vcf1 = test_vcf_1
+    args.region = ""
+    args.vcftype1 = 'hipstr'
+    args.vcf2 = test_vcf_2
+    args.vcftype2 = 'hipstr'
+    retcode = main(args)
+    assert retcode == 0
+    with open(tmpdir + "/test_compare-locuscompare.tab", "r") as out_overall:
+        lines = out_overall.readlines()
+        # Two of the records wont be compared
+        assert len(lines) == 2
+        _, err = capfd.readouterr()
+                        ## first output is about records that have the same starting position but different end pos
+        assert err == ("Records STR_40 and STR_40 overlap:\n"
```

Comment:
For both of these records, the start of the records are the same (if you properly ignore the flanking bps). Can we have one where the starts are different but the ends are the same? It should still say that they overlap but are not comparable.